CONTRIBUTING.md

```diff
@@ -67,6 +67,10 @@ When contributing or amending a set of websites for an App ID, you should state
 
 When contributing or amending a set of websites that require that the user append a generated code to their password when signing in, you should state why you believe the relevant domains require such. This may involve citing a URL to the relevant support page for the website.
 
+### Contributing to Websites That When Embedded as Third-party Ask for Credentials for Other Services
+
+When contributing or amending the list of websites that when embedded as a third party, are known to ask for credentials for other services, you should provide evidence that the given website or websites behaves this way. This may involve a screenshot or steps to navigate a website to observe a subframe behaving this way.
```

Comment:
🤔

It would be nice if you could expand on "when embedded as a third party" and "known to ask for credentials for other services". I'm finding it a little hard to understand what belongs in this new quirks json.

An example from plaid.com in this PR's description would be a good reference for people in the future as an "evidence that the given website or websites behaves this way".

Reply:
Thanks for the feedback! I expanded this out a bit.

Reply:
Thank you for expanding on it :)